plugins/org.obeonetwork.m2doc.doc/doc/user/M2Doc User Guide.html

```diff
@@ -173,6 +173,17 @@ <h4 id="Siriustableproviders">Sirius table providers</h4>
 			<b>Warning</b>: do not forget the single quotes ' around the string in an AQL Expression: 
 			<code>title:"'Table name'"</code>. The description ID does not need such quotes because it is of type String.
 		</p>
+		<h4 id="UpdatestrategyforSiriusRepresentation">Update strategy for Sirius Representation</h4>
+		<p>In both case Sirius diagram and Sirius table providers, the M2Doc generation can occur with a closed Sirius representation view. So the semantic and graphical models could be unsynchronized. A parameter in genConf model define the strategy to follow during M2Doc generation.</p>
+		<p>
+			<img border="0" src="images/image019.png"/>
+		</p>
+		<p>The boolean parameter &#171;Refresh Diagram&#187; can be used to define following update strategy : 
```

Comment:
Is the parameter really named "Refresh Diagram" with a space ? This looks like a label to me. I think we should use IDs, something like 'refreshDiagram'. Additionally, I think it should not mention 'diagrams' but 'representations' since it's an option that is active for both tables and diagrams. Hence 'refreshRepresentations' seems the best name to me .